src/ape/types/__init__.py

```diff
@@ -264,6 +267,19 @@ def get(self, item: str, default: Optional[Any] = None) -> Any:
         return self.event_arguments.get(item, default)
 
 
+class ContractLogContainer(list):
+    __test__ = False
```

Comment:
What's this for?

Reply:
The only time I have used this is when my class name starts with the word `Test` and I don't want `pytest` picking it up.

Reply:
removed test part.

Reply:
> The only time I have used this is when my class name starts with the word `Test` and I don't want `pytest` picking it up, such as `TestAccountContainer`.

Ahhhhh, so that's how you do it

Reply:
I think some stuff in `ape_test` needs this